cms/djangoapps/contentstore/views/transcript_settings.py

```diff
@@ -241,7 +241,7 @@ def transcript_upload_handler(request):
                 content=transcript_file.read().decode('utf-8'),
                 input_format=Transcript.SRT,
                 output_format=Transcript.SJSON
-            )
+            ).encode()
```

Comment:
I'd recommend being explicit about the encoding this to avoid surprises.

```suggestion
            ).encode('utf-8')
```

Reply:
@OmarIthawi We can do it, but we introduce a diff, what do you think?

Reply:
I'm not sure what do you mean. So never mind let's test it on staging.

Reply:
I get it now what do you mean by `diff`. That was a cherry-pick! Makes total sense now.